_episodes/10-publication.md

```diff
@@ -12,6 +12,136 @@ keypoints:
+- Open online repository -- CRAN, GitHub
+- Collaborative, open science tool with ongoing development
+
+What can be released:
+
+
+- **Open Data**: Documenting and sharing research data openly for re-use.
+- **Open Source Software**: Documenting research code and routines, and making them freely accessible and available.
+- **Open Hardware**: Documenting designs, materials, and other relevant information related to hardware, and making them freely accessible and available.
+- **Open Access**: Making all published outputs freely accessible for maximum use and impact.
+- **Open Notebooks**: An emerging practice, documenting and sharing the experimental process of trial and error.
+> https://the-turing-way.netlify.app/reproducible-research/open.html
+
+## Open or Private?
+
+Researchers often worry that they need to hide their code to prevent others stealing it. 
+
+> “After giving talks about open science I’ve sometimes been approached by skeptics who say, ‘Why would I help out my competitors by sharing ideas and data on these new websites? Isn’t that just inviting other people to steal my data, or to scoop me? Only someone naive could think this will ever be widespread.’ As things currently stand, there’s a lot of truth to this point of view. But it’s also important to understand its limits. What these skeptics forget is that they already freely share their ideas and discoveries, whenever they publish papers describing their own scientific work. They’re so stuck inside the citation-measurement-reward system for papers that they view it as a natural law, and forget that it’s socially constructed. It’s an agreement. And because it’s a social agreement, that agreement can be changed. All that’s needed for open science to succeed is for the sharing of scientific knowledge in new media to carry the same kind of cachet that papers do today”
+>
+> Nielsen, M. Reinventing Discovery: The New Era of Networked Science. Princeton University Press, 2011.
+
+https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000246
+
+## Code release
+
+For computational projects, releasing your work in an open repository has parallels with publications. 
+
+<img src="https://i.imgur.com/I9EKxdF.png" alt="drawing" width="800"/> 
+
+There can be specific requirements to keep code bases and/or data private. See the section below for good and not so good reasons for keeping work private.
+
+You can release code and data associated with a research article as a series of files/folders. If your project follows the folder template introduced in a previous episode, for example:
+<img src="https://github.com/jcolomb/gintonic-blog/raw/forreview/50_figures/pulication_ready_figures/2draft-templates.png" alt="drawing" width="300"/> 
+
+> Examples of a template folder tree for a computational project. https://github.com/tonic-team/Tonic-Research-Project-Template
+
+You could bundle folders into a `.zip` file and upload it to Zenodo.
```

Comment:
Maybe add in - You can also link your github account to zenodo, which allows you to push releases of different versions of your repository to Zenodo for long term storage/archiving and to get a digital object identifier (DOI).